validator/client/validator.go

```diff
@@ -667,14 +671,18 @@ func (v *validator) subscribeToSubnets(ctx context.Context, res *ethpb.DutiesRes
 			subscribeSlots = append(subscribeSlots, attesterSlot)
 			subscribeCommitteeIndices = append(subscribeCommitteeIndices, committeeIndex)
 			subscribeIsAggregator = append(subscribeIsAggregator, aggregator)
+			subscribeValidatorIndices = append(subscribeValidatorIndices, validatorIndex)
```

Comment:
cool so this is now collecting indicies for both current and next epochs, I saw a check on our old gRPC that tries to collect a list of indicies based on head and their respective epoch with this I think we just collect both here.